src/main/resources/build/last_rails-osx-association.properties

```diff
@@ -0,0 +1,4 @@
+extension=last_rails
+mime-type=application/rails-18xx-last
+icon=./src/main/resources/images/icon/last_rails.icns
+description=Rails polling file
```

Comment:
I don't think these files (including the icon file(s)) belong in the `src/main/resources` directory. My reason is that these files are not part of the application sources. Instead they are part of the build configuration. Therefore I put the in another folder on the root level to reflect this.

Reply:
And yet they don't belong at the root either :) I think the assumption is that `src/main/resources` are only application "included" resources which isn't really correct, they can be anything used for the application, even for building the final apps (as often there is filtering/templating involved). I've seen some cases where these types of resources are put into something like `src/main/build/` but am on the fence about that as there is some expectation based on the maven nomenclature and it's opinionated standards that tree is well defined given how it's plugins have expectations (such as filtering is out of `src/main/resources/`, etc. In additionally, there were already existing icon, icns, png images in `src/main/resources/images/` and copying them elsewhere lead to duplication which can lead to error proneness (TM on the word I just invented!)

Reply:
I agree that we shouldn't add redundancy, but were we using the icons in the Rails application at all? I'm fine with adding them to the `src` folder, but if we only use them for the installers then `main/resources` does not fit in my opinion. Maybe we should add them to `src/installers` or something similar?

Reply:
I didn't/haven't looked to see if/where they may be used. I did glance at the files I removed (ie for the old installer approach) and didn't see any references there so I assumed they were in use elsewhere. Didn't bother looking deeper than that.